src/hotspot/os/windows/gc/x/xMapper_windows.cpp

```diff
@@ -250,7 +250,7 @@ void XMapper::unreserve_for_shared_awe(uintptr_t addr, size_t size) {
     );
 
   if (!res) {
-    fatal("Failed to unreserve memory: " PTR_FORMAT " " SIZE_FORMAT "M (%d)",
+    fatal("Failed to unreserve memory: " INTPTR_FORMAT " " SIZE_FORMAT "M (%ld)",
```

Comment:
Here, and in many other places:

PTR_FORMAT -> INTPTR_FORMAT is pointless, since both macros are the same. 

We use PTR_FORMAT in many many places to print pointers. I prefer it to INTPTR_FORMAT, since it does not convey the assumption of a type. After all, the type does not matter: we just print the pointer as a raw hex value. Also, INTPTR suggests intptr_t which suggests signedness, which has no place here.

*If* we want to change this, we should first agree on the INTPTR_FORMAT vs PTR_FORMAT difference. And why we use intptr_t in many places that actually call for an uintptr_t.

But I would not change it. There is no need.

Reply:
Yes. Please keep using PTR_FORMAT to print what's semantically a pointer in ZGC.